staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -1927,6 +1927,9 @@ func DescribeProbe(probe *corev1.Probe) string {
 		return fmt.Sprintf("http-get %s %s", url.String(), attrs)
 	case probe.TCPSocket != nil:
 		return fmt.Sprintf("tcp-socket %s:%s %s", probe.TCPSocket.Host, probe.TCPSocket.Port.String(), attrs)
+
+	case probe.GRPC != nil:
+		return fmt.Sprintf("grpc %s:%d %s %s", *(probe.GRPC.Host), probe.GRPC.Port, *(probe.GRPC.Service), attrs)
```

Comment:
hm, why is it `*(probe.GRPC.Host)`, not `probe.GRPC.Host`?

Reply:
follow this review https://github.com/kubernetes/kubernetes/pull/102162#discussion_r683625160